jakartams/src/main/scala/org/apache/pekko/stream/connectors/jakartams/ConnectionRetrySettings.scala

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+/*
+ * Copyright (C) since 2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package org.apache.pekko.stream.connectors.jakartams
+
+import com.typesafe.config.Config
+import org.apache.pekko
+import org.apache.pekko.actor.{ ActorSystem, ClassicActorSystemProvider }
```

Comment:
Not a big deal but if you `import org.apache.pekko`, can the other pekko imports omit the `org.apache` bit?

I won't block a merge over this.